diff --git a/node_modules/@aws-amplify/auth/lib/Auth.d.ts b/node_modules/@aws-amplify/auth/lib/Auth.d.ts
index de51d9c..1fb944b 100644
--- a/node_modules/@aws-amplify/auth/lib/Auth.d.ts
+++ b/node_modules/@aws-amplify/auth/lib/Auth.d.ts
@@ -3,6 +3,7 @@ import { ICredentials } from '@aws-amplify/core';
 import { ISignUpResult, CognitoUser, MFAOption, CognitoUserSession, CognitoUserAttribute, NodeCallback } from 'amazon-cognito-identity-js';
 import { AuthError } from './Errors';
 import { IAuthDevice } from './types/Auth';
+
 /**
  * Provide authentication steps
  */
diff --git a/node_modules/@aws-amplify/auth/src/types/Auth.ts b/node_modules/@aws-amplify/auth/src/types/Auth.ts
index 6b90c5e..5daa211 100644
--- a/node_modules/@aws-amplify/auth/src/types/Auth.ts
+++ b/node_modules/@aws-amplify/auth/src/types/Auth.ts
@@ -28,9 +28,9 @@ export interface SignUpParams {
 }
 
 export interface AuthCache {
-	setItem();
-	getItem();
-	removeItem();
+	setItem(): any;
+	getItem(): any;
+	removeItem(): any;
 }
 
 /**
